sass/_component_search.scss

```diff
@@ -5,14 +5,9 @@
  */
 
 .sidebar-container {
-  @extend .border-right;
-
-  border-right-style: hidden; // hide by default on small devices
-
   /* stylelint-disable-next-line scss/media-feature-value-dollar-variable */
   @media screen and (min-width: map.get($grid-breakpoints, "lg")) {
```

Comment:
I'd prefer we left this in - it was a key part of the original designs to be adopted. 

Maybe this is best discussed as a new issue - just to get some other feedback.

Reply:
Restored

Reply:
thanks